src/components/game-page/rockscissorspaper-page/Timer.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import RockScissorsPaper from './RockScissorsPaper';
+
+const Timer = () => {
+  const [timer, setTimer] = useState(3);
```

Comment:
```suggestion
const [timer, setTimer] = useState(initialValue);
```
timer의 초기값은 props로 받아와도 좋을 것 같아요

Reply:
늦게 봐서 죄송합니다. 😭 
확실히 나중에 3이 아니라 기획이 변경되어서 다른 숫자부터 시작한다고 했을때 더 좋겠네요 좋은 코멘트 감사합니다!!!!!!!!!